### PR TITLE
feat(Theme): tertiary button

### PR DESCRIPTION
**Pull request by jillyan**
**What is the problem this PR is trying to solve?**
TCCUI-65 (Date Picker Improvement) Add a Tertiary button.
**What is the chosen solution to this problem?**
Need one more single action as Tertiary button.
**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [x] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

**Comment by mgirault** [comment]
There is a misunderstanding about naming.
Btn-tertiary-primary, btn-tertiary-secondary are not a good idea,  it was more a reflection phase. :)

We must keep the current approach, the reverse button when the button is gray.
St tropaz color -> "btn btn-tertiary"
Dark silver color -> "btn btn-tertiary btn-inverse"

Yes it is a good idea to put a new tertiary section at the bottom of the page

File: packages/theme/src/theme/_buttons.scss [hunk 1]
```diff
@@ -36,13 +36,16 @@
 	&-success,
 	&-info,
 	&-warning,
-	&-danger {
+	&-danger,
+	&-tertiay-action,
+	&-tertiay-info {
```
Comment (jsomsanith-tlnd) [inline packages/theme/src/theme/_buttons.scss]: You can remove those 2 selectors

File: packages/theme/src/theme/_buttons.scss [hunk 2]
```diff
@@ -59,6 +62,7 @@
 	&-info {
 		@include btn-colors($btn-info-bg);
 		@include btn-inverse-colors($btn-info-bg);
+		@include btn-tertiary-colors($btn-tertiary-info-color);
```
Comment (jsomsanith-tlnd) [inline packages/theme/src/theme/_buttons.scss]:
```suggestion
		@include btn-tertiary-colors($btn-info-bg);
```
Donc define another variable for the same info color. Use the same as the other buttons

File: packages/theme/src/theme/_variables.scss [hunk 3]
```diff
@@ -167,6 +167,11 @@ $btn-secondary-color: #fff !default;
 $btn-secondary-bg: $brand-secondary !default;
 $btn-secondary-border: darken($btn-secondary-bg, 5%) !default;
 
+$btn-tertiary-info-color: $st-tropaz !default;
+$btn-tertiary-default-color: $dark-silver !default;
```
Comment (jsomsanith-tlnd) [inline packages/theme/src/theme/_variables.scss]: You can remove those 2 variables. The colors are not defined specifically for tertiary but for all buttons

File: packages/theme/src/theme/_variables.scss [hunk 4]
```diff
@@ -167,6 +167,11 @@ $btn-secondary-color: #fff !default;
...
+$btn-tertiary-line-height: 2.5rem !default;
+$btn-tertiary-font-weight: 600 !default;
```
Comment (jsomsanith-tlnd) [inline packages/theme/src/theme/_variables.scss]: This is not in the guideline, can you check with the UX team to get the wanted value and for them to add it in the guideline please ?

File: packages/theme/src/theme/_visual-helpers.scss [hunk 5]
```diff
@@ -176,3 +176,27 @@
 		}
 	}
 }
+
+@mixin btn-tertiary-colors($color: $btn-tertiary-default-color) {
```
Comment (jsomsanith-tlnd) [inline packages/theme/src/theme/_visual-helpers.scss]:
```suggestion
@mixin btn-tertiary-colors($color: $btn-default-bg) {
```

File: packages/theme/src/theme/_visual-helpers.scss [hunk 6]
```diff
@@ -176,3 +176,27 @@
...
+		transition: background 0.2s ease-in;
+		color: $color;
+		&:hover {
+			color: $color;
+			background: tint($color, 90%);
+		}
+		&:active {
+			background: tint($color, 80%);
+		}
+		&:focus {
```
Comment (jsomsanith-tlnd) [inline packages/theme/src/theme/_visual-helpers.scss]: Focus should have the same style as hover (accessibility)

File: packages/theme/src/theme/_variables.scss (hunk 4 above)
Comment (jillyan) [reply packages/theme/src/theme/_variables.scss]: height: 25px is in guideline.
font weight: 600 is missed in guideline. Mickaël G sent this to me in slack.

Comment (jillyan) [reply packages/theme/src/theme/_variables.scss]: I will request ux to add this font weight.

File: packages/theme/src/theme/_buttons.scss (hunk 1 above)
Comment (jillyan) [reply packages/theme/src/theme/_buttons.scss]: updated

File: packages/theme/src/theme/_buttons.scss (hunk 2 above)
Comment (jillyan) [reply packages/theme/src/theme/_buttons.scss]: updated

File: packages/theme/src/theme/_variables.scss (hunk 3 above)
Comment (jillyan) [reply packages/theme/src/theme/_variables.scss]: updated

File: packages/theme/src/theme/_visual-helpers.scss (hunk 5 above)
Comment (jillyan) [reply packages/theme/src/theme/_visual-helpers.scss]: updated

File: packages/theme/src/theme/_visual-helpers.scss (hunk 6 above)
Comment (jillyan) [reply packages/theme/src/theme/_visual-helpers.scss]: updated

File: packages/theme/src/theme/_variables.scss (hunk 4 above)
Comment (jsomsanith-tlnd) [reply packages/theme/src/theme/_variables.scss]: 👍  I added it in the guideline then

**Pull request merged**